arviz/plots/distcomparisonplot.py

```diff
@@ -130,9 +131,17 @@ def plot_dist_comparison(
     len_plots = rcParams["plot.max_subplots"] // (len(groups) + 1)
     len_plots = len_plots if len_plots else 1
     dc_plotters = [
-        list(xarray_var_iter(data, var_names=var, combined=True))[:len_plots]
+        list(xarray_var_iter(data, var_names=var, combined=True)) [:len_plots]
```

Comment:
It looks like it is being truncated twice now, so I think the warning will always be triggered. Can you check locally in a few cases in addition to the test?

Reply:
Yes, I will look into it

Reply:
Should I alter the .gitignore?

Reply:
Yes, adding things to the gitignore is fine, removing them is not